tests/core/scenario/test_scenario_manager.py

```diff
@@ -203,10 +203,14 @@ def test_create_and_delete_scenario():
     assert scenario_1.properties["name"] == name_1
     assert scenario_1.tags == set()
 
-    with pytest.raises(DeletingPrimaryScenario):
-        _ScenarioManager._delete(
-            scenario_1.id,
-        )
+    cycle_id_1 = scenario_1.cycle.id
+    assert _CycleManager._get(cycle_id_1).id == cycle_id_1
```

Comment:
can you add another test about raising DeletingPrimaryScenario when the cycle has more than 1 scenario?

Reply:
It is already in line 230.

Reply:
Ah! Saw it thanks!